app/scripts/move-popup/dimTalentGrid.directive.js

```diff
@@ -66,5 +66,12 @@
       vm.numColumns = _.max(vm.talentGrid.nodes, 'column').column + 1;
       vm.numRows = _.max(vm.talentGrid.nodes, 'row').row + 1;
     }
+
+    vm.onScopeClick = function(node) {      
+      if (node.column == 1) {
```

Comment:
we should restrict this to weapons only. something like: `node.column === 1 && node.primStat.statHash === 368428387`